packages/InteractiveProfilingTool-Core.package/IPTReport.class/class/openReportFor..st

```diff
@@ -4,10 +4,12 @@ openReportFor: aMessageTally
 	| tree root userCallsRoot |
 	
 	root := self constructWrappedTreeWithRoot: aMessageTally.
+	
 	userCallsRoot := root findRootForUserCalls.
 	
 	tree := self new
-		initializeWithRootNode: userCallsRoot;
+		initializeWithUserCallsRootsNode: userCallsRoot;
+		initializeWithRootNode: root;
```

Comment:
The naming of the two functions is not optimal in my opinion. I would assume that an object is initialized once. Maybe change it to `setUserCallsRootNode` and `setRootNode`.

Reply:
✅ We introduced a new class method `IPTReport >> withRoot:withUserCallsRoots:`.